rosidl_generator_py/resource/_msg_support.c.em

```diff
@@ -84,7 +84,11 @@ nested_type = '%s__%s__%s' % (field.type.pkg_name, 'msg', field.type.type)
   @(nested_type) * dest@(field.name) = ros_message->@(field.name).data;
 @[      else]@
   size_t size@(field.name) = @(field.type.array_size);
+@[        if field.type.is_upper_bound]@
+  @(nested_type) * dest@(field.name) = ros_message->@(field.name).data;
+@[        else]@
   @(nested_type) * dest@(field.name) = ros_message->@(field.name);
+@[        end if]@
```

Comment:
Alright, in this case let's merge the first iteration and get things fixed before actually merging ros2/system_tests#124